script/add-raml

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+set -eou pipefail
+
+ROOT=$(cd $(dirname $0)/..; pwd)
+source ${ROOT}/env
+source ${ROOT}/script/include/setup.sh
+
+STAGING=
+
+if [ ${1:-} = "--stage" ] || [ ${1:-} = "-s" ]; then
+  STAGING=yes
+  CONTENT_URL=${STAGING_CONTENT_URL}
+  shift
+else
+  CONTENT_URL=${PROD_CONTENT_URL}
+fi
+
+if [ -z "${1:-}" ]; then
+  cat <<EOM 1>&2
+Usage: script/add-raml [--stage|-s] <path>
+
+ --stage|-s - Submit content to the staging endpoint.
+ path - Path to a Raml file.
```

Comment:
This path is to the content repository hosting RAML files, actually. So:
```shell
path - Path to a RAML content repository
```

Reply:
Updated